### PR TITLE
Require category at least one

### PR DESCRIPTION
**Pull request by tricknotes**

### CODE REVIEW

**Comment by tricknotes** [comment]
Ouch!
We should show error message on post form. -> WIP

**Comment by tricknotes** [comment]
> We should show error message on post form.

done

File: app/views/editor/posts/_form.html.haml [hunk 1]
```diff
@@ -27,6 +27,9 @@
     = f.fields_for :categorizations do |ff|
       = render 'categorization_fields', f: ff
 
+    %span{class: f.send(:has_error?, :categorizations) ? 'has-error' : nil}
+      = f.send :generate_help, :categorizations, nil
+
```
Comment (okkez) [inline app/views/editor/posts/_form.html.haml:32]: https://github.com/bm-sms/daimon-news/blob/c52f985b0478a836303ad2fe730e34e50e4b119b/app/validators/post_category_validator.rb#L5

Can we display error message if we change key on the above code?

This comment is just question.

**Comment by okkez** [comment]
Looks awesome!

File: app/views/editor/posts/_form.html.haml (hunk 1 above)
Comment (tricknotes) [inline app/views/editor/posts/_form.html.haml:32]: yes!

**Pull request merged**